modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/MultiMapUtils.java

```diff
@@ -0,0 +1,72 @@
+package de.wuespace.telestion.api.message;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.eventbus.DeliveryOptions;
+import io.vertx.core.eventbus.Message;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Utilities for the {@link MultiMap Vert.x multimap}.
+ *
+ * @author Ludwig Richter (fussel178)
+ */
+public class MultiMapUtils {
+
+	/**
+	 * Merges an array of multi-maps into one multimap
+	 * with the {@link MultiMap#addAll(MultiMap) addAll} function.
+	 *
+	 * @param maps multiple multi-maps that should be merged into one
+	 * @return a new multimap with the merged content of the other multi-maps
+	 */
```

Comment:
It's either with a dash or without it. We can't have both 😉 

```suggestion
	/**
	 * Merges an array of multi-maps into one multi-map
	 * with the {@link MultiMap#addAll(MultiMap) addAll} function.
	 *
	 * @param maps multi-maps that you want to merge
	 * @return a new multi-map with the merged content of the other multi-maps
	 */
```